docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Material-UI is now MUI
+description: Starting today, we are evolving our brand identity. We are clarifying the difference between our company and our products.
+date: 2021-09-15T00:00:00.000Z
+authors: ['oliviertassinari', 'danilo-leal', 'mbrookes']
+card: true
+---
+
+Starting today, we are evolving our brand identity. We are clarifying the difference between our company and our products.
+
+- Material-UI: the organization is now called [**MUI**](https://github.com/mui-org/).
+- Material-UI: the set of foundational MIT React components is now called [**MUI Core**](https://github.com/mui-org/material-ui).
+- Material-UI X: the set of advanced React components is now called [**MUI X**](https://github.com/mui-org/material-ui-x).
+
+The new identity matches with the increased scope of our focus.
+
+## The origin
+
+Material-UI started as a React implementation of the Material Design guidelines in 2014.
+The goal was simple, to empower the React developers to use Material Design.
+The community [loved it](https://news.ycombinator.com/item?id=8582439) and was craving for more, so we kept building.
+
+We have made it smooth for the community to contribute.
+We started actively reviewing pull requests and guiding issues.
+Things started to really take off with [the release of v1](/blog/material-ui-v1-is-out/) in 2018.
+The frontend community was actively migrating to React, in desperate need to transition quickly,
```

Comment:
Really?

Reply:
The underlying flow of thoughts: when you migrate your application from jQuery to React, you don't have the time to build custom components. Would you agree with it? We can rephrase

Reply:
```suggestion
The frontend community was actively migrating to React, but didn't have the bandwidth to rebuild legacy interfaces from scratch,
```